plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcMetricsCollection.java

```diff
@@ -249,6 +255,41 @@ public void testNestedTypes()
         assertUpdate("DROP TABLE test_nested_types");
     }
 
+    @Test
+    public void testWithTimestamps()
```

Comment:
Does this gives more coverage for ORC+timestamps than `io.trino.plugin.iceberg.BaseIcebergConnectorTest#testSelectByTimestamp`?

Reply:
This adds test coverage of min()/max() functions on timestamp columns. If we ever made an optimization to use file-level statistics for min/max function then we will be notified.